### PR TITLE
Add LibreSelery

### PR DESCRIPTION
**Pull request by Ly0n**
We created a Coinbase API based tool that allows you to distribute funding with you contributor and dependencies.
This new concept is called "Continuous Funding". At the moment we just support BTC but we will extend into other cryptocurrencies. 
 
We use our own tool for developing LibreSelery:

Find more information on our blog post:
https://protontypes.eu/blog/2020/09/02/launch-of-protontypes/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ A curated list of bitcoin services and tools for software developers
 * [pycoin](https://github.com/richardkiss/pycoin) - Python library for Bitcoin keys, signatures, transactions. Includes full VM implementation and tools for manipulating keys (ku) and transactions (tx).
 * [bitcoin_tools](https://github.com/sr-gi/bitcoin_tools) - Python library for building and analyzing transactions and scripts (both standard and custom). Comes along with a UTXO set analysis tool. Includes several examples and exhaustive documentation.
 * [oogway](https://github.com/merwane/oogway) - A secure Bitcoin utility library for Python.
+* [LibreSelery] - Continuous funding distribution to your project contributors and dependencies with cryptocurrency
```
Comment (Overtorment) [inline README.md:101]: wheres the link?

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ly0n) [reply README.md]: Sorry. A stupid mistake. Here the link.

**Review by Overtorment — CHANGES_REQUESTED** [review]
Needs to be in a different section. Probably even add new section for projects like this..?
This is clearly not just "python library" for bitcoin developers

**Comment by Overtorment** [comment]
How about "Tools & services"?

**Comment by Overtorment** [comment]
I vote for "Utilities" then. Put it there.

**Comment by Ly0n** [comment]
It is now in utilities. Thanks for the fast feedback.

**Pull request merged**